CHANGELOG.md

```diff
@@ -34,12 +37,61 @@ All notable changes to this project will be documented in this file.
+  - Added by [Jon Shier](https://github.com/jshier) in Pull Request [#2346](https://github.com/Alamofire/Alamofire/pull/2346).
+- `HTTPHeaders` type and convenience APIs.
+  - Added by [Jon Shier](https://github.com/jshier) in Pull Request [#2629](https://github.com/Alamofire/Alamofire/pull/2629).
+- Support for `Encodable` parameter types with `JSONParameterEncoder` and `URLEncodedFormParameterEncoder`.
+  - Added by [Jon Shier](https://github.com/jshier) in Pull Request [#2637](https://github.com/Alamofire/Alamofire/pull/2637).
+- Customizable empty reponse handling in response serializers.
+  - Added by [Jon Shier](https://github.com/jshier) in Pull Request [#2639](https://github.com/Alamofire/Alamofire/pull/2639).
+- `EventMonitor` protocol, to tap in to internal Alamofire lifetime events.
+  - Added by [Jon Shier](https://github.com/jshier) in Pull Request [#2521](https://github.com/Alamofire/Alamofire/pull/2521).
+- `ServerTrustFailureReason` to `AFError`.
+  - Added by [Jon Shier](https://github.com/jshier) in Pull Request [#2608](https://github.com/Alamofire/Alamofire/pull/2608). 
+- Asynchronous request creation.
+  - Added by [Jon Shier](https://github.com/jshier) in Pull Request [#2521](https://github.com/Alamofire/Alamofire/pull/2521).
+  
+#### Updated
+- `ResponseSerializer` and associated protocols' requirements and implementations.
+  - Updated by  [Jon Shier](https://github.com/jshier) in Pull Requests [#2265](https://github.com/Alamofire/Alamofire/pull/2265)  and [#2639](https://github.com/Alamofire/Alamofire/pull/2639).
+- `RequestAdapter` to allow for asynchronous adaptation.
+  - Updated by  [Jon Shier](https://github.com/jshier) in Pull Request [#2628](https://github.com/Alamofire/Alamofire/pull/2628) .
+- `SessionManager` to `Session`, with rewritten implementation and API.
+  - Updated by [Jon Shier](https://github.com/jshier) in Pull Request [#2521](https://github.com/Alamofire/Alamofire/pull/2521).
+- `Request`, `DataRequest`, `UploadRequest`, and `DownloadRequest` with rewritten implementation and API.
+  - Updated by [Jon Shier](https://github.com/jshier) in Pull Request [#2521](https://github.com/Alamofire/Alamofire/pull/2521).
+- Top level `request` API to be inside a real `AF` namespace enum rather than unnecessary `Alamofire.` usage.
+   - Updated by [Jon Shier](https://github.com/jshier) in Pull Request [#2585](https://github.com/Alamofire/Alamofire/pull/2585).
+- Internal implementation of `RequestRetrier`.
+   - Updated by [Jon Shier](https://github.com/jshier) in Pull Request [#2521](https://github.com/Alamofire/Alamofire/pull/2521).
+- Internal use of  `DispatchQueue`.
+   - Updated by [Jon Shier](https://github.com/jshier) in Pull Request [#2521](https://github.com/Alamofire/Alamofire/pull/2521).
+- Internal locking API.
+   - Updated by [Jon Shier](https://github.com/jshier) in Pull Request [#2290](https://github.com/Alamofire/Alamofire/pull/2290).
```

Comment:
These three entries should be more specific or removed. If they don't affect the end user in any way, then they should be written with specifics for internal developers. Otherwise I don't think they serve much point.

Reply:
You're right, it was mainly to make people aware that behaviors they'd see around those aspects of the library have changed. I've removed them.

Reply:
Removed.